modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -170,15 +171,34 @@ Because the following endpoints use mutual TLS authentication to establish the c
 .2+| 443 .2+| HTTPS | US control plane: `ecr.us-east-1.amazonaws.com` | Runtime Fabric Docker repository.
 | EU control plane: `ecr.eu-central-1.amazonaws.com` | Runtime Fabric Docker repository.
 .2+| 443 .2+| HTTPS | US control plane: `494141260463.dkr.ecr.us-east-1.amazonaws.com` | Runtime Fabric Docker repository.
-| EU control plane: `494141260463.dkr.ecr.eu-central-1.amazonaws.com` | Runtime Fabric Docker repository.
+| EU control plane: `655988475869.dkr.ecr.eu-central-1.amazonaws.com` | Runtime Fabric Docker repository.
 .2+| 443 .2+| HTTPS | US control plane: `api.ecr.us-east-1.amazonaws.com` | Runtime Fabric Docker repository.
 | EU control plane: `api.ecr.eu-central-1.amazonaws.com` | Runtime Fabric Docker repository.
 .2+| 443 .2+| HTTPS | US control plane: `prod-us-east-1-starport-layer-bucket.s3.amazonaws.com` `prod-us-east-1-starport-layer-bucket.s3.us-east-1.amazonaws.com` | Runtime Fabric Docker image delivery.
 | EU control plane: `prod-eu-central-1-starport-layer-bucket.s3.amazonaws.com` `prod-eu-central-1-starport-layer-bucket.s3.eu-central-1.amazonaws.com` | Runtime Fabric Docker image delivery.
 .2+| 443 .2+| HTTPS | US control plane: `runtime-fabric.s3.amazonaws.com` | Runtime Fabric Docker repository.
 | EU control plane: `runtime-fabric-eu.s3.amazonaws.com` | Runtime Fabric Docker repository.
+.2+| 443 .2+| HTTPS | US control plane: `configuration-resolver.prod.cloudhub.io` | Anypoint Configuration Resolver.
+| EU control plane: `configuration-resolver.prod-eu.msap.io` | Anypoint Configuration Resolver.
 |===
 
+=== Mutual Authentication with Anypoint Platform
+
+On Runtime Fabric instances running version 1.4.x and later, you can use the Runtime Manager *Settings* page to enable mutual 
+authentication for the cluster. Enabling mutual authentication provides better security for communications between the Runtime Fabric cluster and Anypoint Platform and is highly recommended.
```

Comment:
Mutual authentication improves the security of communication between Runtime Fabric and Anypoint Platform, and is highly recommended.